pkgs/development/libraries/science/math/nccl/default.nix

```diff
@@ -1,11 +1,19 @@
-{ lib, stdenv, fetchFromGitHub, which, cudaPackages, addOpenGLRunpath }:
+{ lib
+, stdenv
```

Comment:
should this also be called `backendStdenv` for consistency?

Reply:
I think this even allows to drop the `NVCC_APPEND_FLAGS` line!